Initial Pull Request

BDesselle opened the pull request:
# Description

Please include a summary of the change and a link to which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

- [ ] Design a templete to base the landing page on.

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

BDesselle committed:
Initial Commit

**Added**
-index.html with a very basic markup structure
-madia/logo.png a placeholder logo
-css/bootstrap.min.css basic bootstrap file
-css/index.css basic css file, no reset or universals

eddygonzalez9708 reviewed (approved):
Great job setting up a basic HTML template for this project. Remember, when creating a PR request you should modify the information in the PR comment and add reviewers from the team so that they can receive an email when you submit a PR.

BDesselle committed:
Rough Navigation

BDesselle committed:
Rough Header

BDesselle committed:
Rough Sections

BDesselle committed:
Refactorizing

Going to be going with a video background and bootstrap cards.

BDesselle committed:
Layout Updated

Layout Updated to be a clear container housing bootstrap cards for the data I plan to implement into the site.
